packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -162,12 +162,13 @@ function makeOperationGenerator(optionsParam?: OperationGenerationConfig): Gener
 
     function inclusiveRange(state: ClientOpState): RangeSpec {
         const start = position(state);
-        const end = state.random.integer(start, state.sharedString.getLength() - 1);
+        const end = state.random.integer(start, Math.max(start, state.sharedString.getLength() - 1));
         return { start, end };
     }
 
     function propertySet(state: ClientOpState): PropertySet {
-        const propNamesShuffled = state.random.shuffle(options.propertyNamePool);
+        const propNamesShuffled = [...options.propertyNamePool];
+        state.random.shuffle(propNamesShuffled);
```

Comment:
@Abe27342 - The 'random-js' shuffle API shuffles the array in place and then returns a reference to the original.

I find this API error-prone, as it tends to lead people to believe the original array is not mutated, so I returned void instead.

From the variable names, I think the intent here was to produce a new array?

Reply:
It ultimately shouldn't matter, but yeah this choice is safer and maybe less disorienting for debugging.